src/validation/rules/OverlappingFieldsCanBeMergedRule.ts

```diff
@@ -469,6 +483,47 @@ function findConflictsBetweenSubSelectionSets(
   return conflicts;
 }
 
+// (E) Then collect any conflicts between the provided collection of fields
+// and any fragment names found in the given fragment.
+const processDiscoveredFragments = (
```

Comment:
Could this be declared as a named function rather than an arrow function?

Reply:
Is there a reason for that? 😅 but yes can do, does this stop approvals though?

Reply:
Just style